docs/services/data/_index.md

```diff
@@ -103,6 +103,18 @@ To configure cloud sync, see [configure cloud sync](../data/configure-cloud-sync
 
   {{< /alert >}}
 
+## Data query
+
+Once you have synced data to the cloud, you can query certain types of data directly in the cloud using MQL, the [MongoDB query language](https://www.mongodb.com/docs/manual/tutorial/query-documents/).
+Synced data is stored in a MongoDB [Atlas Data Federation](https://www.mongodb.com/docs/atlas/data-federation/overview/) instance.
+
+You can query both the captured data itself as well as its metadata (such as robot ID, organization ID, and [tags](/manage/data/label/#image-tags)).
+
+In order to query data using MQL, you must have first [captured data](/services/data/configure-data-capture/) and [synced data](/services/data/configure-cloud-sync/) to the cloud.
+You must also configure a new database connection to the Viam cloud data store using the Viam CLI.
```

Comment:
This will soon not be true — this flow will only be if you want to use a database connection to interfacing with MongoDB directly.

We have just added a [TabularDataBySQL and TabularDataByMQL](https://github.com/viamrobotics/api/blob/020985a15c113694e4589f725813b9077b6050b0/proto/viam/app/data/v1/data.proto#L189-L214) API which will also be available in the UI within the next few days. Scope doc was added last week, and implementation was just finished — added the link into https://viam.atlassian.net/browse/DOCS-1335. That will now be the intended main flow; since the user has already authenticated with Viam, they can now just use the Viam API to make an SQL/MQL query and get a list of results back. Deployment: https://pr-2919-appmain-bplesliplq-uc.a.run.app/data/query